src/bundle/SystemInfo/Collector/IbexaSystemInfoCollector.php

```diff
@@ -129,56 +152,58 @@ public function __construct(SystemInfoCollector $composerCollector, $debug = fal
      */
     public function collect(): IbexaSystemInfo
     {
-        $ibexa = new IbexaSystemInfo(['debug' => $this->debug, 'composerInfo' => $this->composerInfo]);
-        if ($this->composerInfo === null) {
-            return $ibexa;
-        }
+        $vendorDir = sprintf('%s/vendor/', $this->kernelProjectDir);
 
+        $ibexa = new IbexaSystemInfo([
+            'debug' => $this->debug,
+            'name' => EzSystemsEzSupportToolsExtension::getNameByPackages($vendorDir),
```

Comment:
In context here we could have reused info in composer info, but code reuse is good ;)

Reply:
> In context here we could have reused info in composer info, but code reuse is good ;)

AFAICS this is not exposed in composer info right now, so would require more changes. I prefer to KISS here for now ;)

Reply:
The package info are there in composerInfo. But problem  is that can't rely on this info being available at compile time which is needed by powered by logic. So this is only way to way able to reuse the naming logic, so you did good ;) 👍